app/src/main/java/com/hedvig/app/feature/keygear/ui/itemdetail/KeyGearItemDetailViewModel.kt

```diff
@@ -34,36 +36,49 @@ class KeyGearItemDetailViewModelImpl(
 
     override fun loadItem(id: String) {
         viewModelScope.launch {
-            runCatching {
+            val result = runCatching {
```

Comment:
We do not need to wrap `Flow`s with `runCatching`, as are already safe when we add a `catch`-clause for them 🙂